modules/benchmark/R/calc.metrics.R

```diff
@@ -12,44 +11,74 @@
 ##' 
 ##' @author Betsy Cowdery
 
-calc.metrics <- function(data.path, format, vars_used, model_run, metrics, start_year, end_year, site){
+calc.metrics <- function(data.path, format, model_run, metrics, start_year, end_year, site, bm, ens){
   
-  # Right now many argument are being read in as R list objects
-  # This won't work remotely but at the moment I don't have a good solution
+  # Right now I'm making the inappropriate assumption that storage type will be 
+  # empty unless it's a time variable. 
+  # This is because I haven't come up for a good way to test that a character string is a date format
```

Comment:
How about you build the met row detection into the format object, so that time.row is generated by the function that queries formats. That way we only do this check in one place and if we decide to change how we detect met rows or store them in the database it doesn't affect any down-stream code (i.e. would only need to be changed in one place).